docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md

```diff
@@ -142,7 +142,22 @@ On December 7th 2023 we set the new Coverage engine to post coverage summaries t
 
 On December 27th 2023 we set the [diff tab of the Coverage pull requests page](../../repositories-coverage/pull-requests.md#diff-tab) to get data from the new Coverage engine.
 
-## Differences in coverage metrics between the old and new Coverage engines
+## Missing data and differences in reported metrics
+
+### Missing data on the Coverage Pull request page {: id="differences-in-coverage-metrics-between-the-coverage-and-quality-pull-request-pages" }
+
+You may notice missing coverage data on the [Coverage Pull requests page](../../repositories-coverage/pull-requests.md) for older pull requests when compared to the coverage data shown on the [Quality Pull requests page](../../repositories/pull-requests.md). This may happen due to a number of reasons:
+
+-   GitHub only: The new Coverage service requires updated app permissions to calculate coverage data. If you haven't done so yet, please [review and accept the updated Codacy app permissions](https://docs.github.com/en/enterprise-cloud@latest/apps/using-github-apps/reviewing-and-modifying-installed-github-apps#reviewing-permissions) on GitHub and re-analyze the pull request by pushing an empty commit.
+
+-   The data may be missing because the latest commit to the pull request was pushed before the new Coverage engine was enabled. To resolve this, please re-analyze the pull request by pushing an empty commit:
```

Comment:
Same here:

```suggestion
-   The data may be missing because the latest commit to the pull request was pushed before the new Coverage engine was enabled. To resolve this, re-analyze the pull request by pushing an empty commit:
```